index.html

```diff
@@ -1,225 +1,240 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio MVOC</title>
-    <link rel="stylesheet" href="styles.css">
-  </head>
-  <body>
-    <header>
-      <div class="greeting">Welcome</div>
-      <nav>
-        <a href="#">
-          <img src="/images/Icon-Menu.png" alt="hamburger-icon">
-        </a>
-      </nav>
-    </header>
-    <main class="main-section">
-      <section class="headline">
-        <h1 class="title name">
-          Hey there. I'm Marco
-        </h1>
-        <h2 class="title ocupation">
-          I'm a software developer
-        </h2>
-        <p>
-          I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
-        </p>
-        <div class="social-container">
-          <ul>
-            <li>
-              <a href="#"><img src="/images/twitter.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/Linkedin-icon.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/medium.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/github.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/angellist.png" alt=""></a>
-            </li>
-          </ul>
-          <a class="down-icon" href="#">
-            <img src="/images/down.png" alt="down-icon">
-          </a>
-        </div>
-      </section>
-      <section class="works">
-        <h2 class="section-title">
-          My Recent Works
-        </h2>
-        <div class="card-container">
-          <div class="card">
-            <div class="card-img">
-            </div>
-            <div class="card-content">
-              <h4 class="p-title"> 
-                Multi-Post Stories Gain+Glory 
-              </h4>
-              <ul class="tags">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavaScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="primary-btn">
-                See Project
-              </button>
-            </div>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Marco Oquendo</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <span class="welcome">Welcome</span>
+    <nav>
+      <a href="#"><img src="/images/Icon-Menu.png" alt="hamburger-icon"></a>
+    </nav>
```

Comment:
- [ ] For desktop navigation, the logo and hamburger button shouldn't be displayed in this screen size

- [ ] Kindly add and center the links 👍🏿 


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_320](https://user-images.githubusercontent.com/69638013/178111677-60efb9b7-965c-4174-82f9-c30ce1ead4a6.png) | ![Screenshot_319](https://user-images.githubusercontent.com/69638013/178111579-be768fa5-e46e-404e-a5ed-24521cbd42ae.png)